### PR TITLE
Add xdebug for devs

### PR DESCRIPTION
**Pull request by ColonelMoutarde**
Changes proposed in this pull request:

- add xdebug in project only for developpeurs

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,13 @@
-FROM alpine:3.18
+FROM alpine:3.18 as dev
```
Comment (Alkarex) [inline .devcontainer/Dockerfile]: Still a single-stage build, right? So no need to have a name

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 
 ENV TZ UTC
 SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
 
 RUN apk add --no-cache \
 	tzdata \
 	apache2 php-apache2 \
+    && pecl install xdebug-3.2.2 \
+    && docker-php-ext-enable xdebug \
```
Comment (Alkarex) [inline .devcontainer/Dockerfile:10]: That does not build

**Review by Alkarex — CHANGES_REQUESTED** [review]
Does not work for multiple reasons. See comments

File: tests/app/Services/ImportServiceTest.php
Comment (Alkarex) [inline tests/app/Services/ImportServiceTest.php:1]: That does not seem to be related to this PR, is it? And fails the tests

Comment (ColonelMoutarde) [reply tests/app/Services/ImportServiceTest.php:1]: Indeed, I don't understand what's going on.

**Pull request closed without merging**